### ISSUE TITLE
Message property in template overwrites Message property in output

### ISSUE THREAD

**Issue by rogeralsing** [serilog-contrib/serilog-sinks-grafana-loki#94]
### Which version of Serilog.Sinks.Grafana.Loki are you using?

v7.10

### Which version of .NET are you using?

net5

### Describe the bug

When using this sink to push to Loki, we get json data like so:

```csharp
logger.LogInfo("Hello {Who}", world);
```

```json
{
     "Message":  "Hello World",
     "MessageTemplate": "Hello {Who}",
     "Who": "World"
}
```

### To Reproduce

But,
If the property is named "Message", you instead get the following output:

```csharp
logger.LogInfo("Hello {Message}", world);
```

```json
{
     "Message":  "World",
     "MessageTemplate": "Hello {Message}",
}
```

The entry for the formatted output has now been replaced with the content of the logged property "Message".

We can work around this by making sure we don't use "Message" as a logging property.
But. in an event-driven architecture, and in such libraries, there are plenty of cases where the property name "Message" is used.

Is there any way to getaround this?

### Expected behavior

Maybe being able to control what the formatted output field is named, e.g. `_message`

```csharp
logger.LogInfo("Hello {Message}", world);
```

```json
{
     "_message":  "Hello World",   //<--- the formatted output
     "MessageTemplate": "Hello {Message}",
     "Message": "World"  //<--- the logging property Message
}
```

### I have read the documentation

- [X] I have read [Common Problems](https://github.com/serilog-contrib/serilog-sinks-grafana-loki/wiki/Common-problems) section in project wiki

**Comment by mishamyte** [comment]
Hi @rogeralsing,

It seems for me that described behavior is caused by Serilog's [formatting](https://github.com/serilog/serilog/wiki/Formatting-Output#formatting-plain-text)

Have u faced this issue for another sinks? I will take a look at this behavior

**Comment by rogeralsing** [comment]
Never seen this in any other sink.
It is only the json output itself that is the issue. which afaik, is generated by this lib (?)
https://github.com/serilog-contrib/serilog-sinks-grafana-loki/blob/a381464898fd17fcd187986e39cec5597bdd0512/src/Serilog.Sinks.Grafana.Loki/LokiJsonTextFormatter.cs#L67-L71

Which is then overwritten by:
https://github.com/serilog-contrib/serilog-sinks-grafana-loki/blob/a381464898fd17fcd187986e39cec5597bdd0512/src/Serilog.Sinks.Grafana.Loki/LokiJsonTextFormatter.cs#L113


If there is a property name called "Message"

Or rather, there will be a Json payload with a duplicate entry of "Message" where Loki only consumes the last occurance

**Comment by mishamyte** [comment]
Thanks, seems like a mess. Will look closer at this case. Unfortunately it is hard to give a estimate, due to Russian military invasion into my country

**Comment by EraYaN** [comment]
I think this is why some other serilog formatters (like `Serilog.Formatting.Compact.RenderedCompactJsonFormatter, Serilog.Formatting.Compact`) use the `@m`, `@x`, `@i` and `@t` (message, exception, instance and timestamp respectively) for the internal values.

**Comment by mishamyte** [comment]
Fixed in [v7.1.1](https://github.com/serilog-contrib/serilog-sinks-grafana-loki/releases/tag/v7.1.1)
